### PR TITLE
feat: event attribute get operation support

### LINKED ISSUE #49: CloudEvent class needs better access to cloudevent fields
## Expected Behavior
```python
event = CloudEvent(headers, data)
event.specversion
```

## Actual Behavior
```python
event = CloudEvent(headers, data)
event['specversion']
```

## Steps to Reproduce the Problem
```python
from cloudevents.sdk import converters
from cloudevents.sdk.http_events import CloudEvent
import requests


attributes = {
    "Content-Type": "application/json",
    "type": "README.sample.binary",
    "id": "binary-event",
    "source": "README",
}
data = {"message": "Hello World!"}

event = CloudEvent(attributes, data)
event['specversion']
```

### PR DESCRIPTION
**Pull request by sasha-tkachev**
Fixes #49

## One line description for the changelog
I implemented a support for the `get` operation on the event attributes as described in #147 

- [v] Tests pass
- [v] Appropriate changes to README are included in PR

### CODE REVIEW

File: cloudevents/http/event.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CloudEvent:
     """
 
     def __init__(
-        self, attributes: typing.Dict[str, str], data: typing.Any = None
+            self, attributes: typing.Dict[str, str], data: typing.Any = None
```
Comment (xSAVIKx) [inline cloudevents/http/event.py:30]: Can you please reformat the code using `black` utility? In particular, the indentation should be fixed by `black` here.

https://github.com/psf/black

File: cloudevents/tests/test_http_cloudevent.py [hunk 2]
```diff
@@ -16,48 +21,46 @@ def test_http_cloudevent_equality(specversion):
...
         "subject": "my-subject",
     }
-    data = '{"name":"john"}'
-    event1 = CloudEvent(attributes, data)
-    event2 = CloudEvent(attributes, data)
+
+
+_my_dummy_data = '{"name":"john"}'
+_your_dummy_data = '{"name":"paul"}'
+assert _my_dummy_data != _your_dummy_data
```
Comment (xSAVIKx) [inline cloudevents/tests/test_http_cloudevent.py:28]: Can you please avoid having any runnable things and assertions in the file scope?

I'd highly recommend migrating these tests to Class-like test cases.

File: cloudevents/http/event.py [hunk 3]
```diff
@@ -77,6 +77,15 @@ def __eq__(self, other):
     def __getitem__(self, key):
         return self._attributes[key]
 
+    def get(self, key: str, default: typing.Optional[typing.Any] = None) -> typing.Optional[typing.Any]:
+        """
+        Get a an attribute value, return default if does not exist.
+        MUST NOT throw exception when the key does not exist
+        :param key: attribute name
+        :param default: value which will be returned if the attribute does not exist
```
Comment (xSAVIKx) [inline cloudevents/http/event.py:85]:
```suggestion
        Retrieves an event attribute value for the given key. Returns the default value if not attribute for the given key exists.
        
        MUST NOT throw an exception when the key does not exist.
        
        :param key: The event attribute name.
        :param default: The default value to be returned when no attribute with the given key exists.
```
Can you please also document the returned value?

**Review by xSAVIKx — COMMENTED** [review]
@sasha-tkachev thx for your PR and the contribution! Can you please address some of the comments noted below?

**Comment by sasha-tkachev** [comment]
@xSAVIKx I have addressed all of your comments :)

File: cloudevents/http/event.py (hunk 3 above)
Comment (xSAVIKx) [reply cloudevents/http/event.py:87]: @sasha-tkachev you've not accepted the suggestion nor fixed the grammar. Please take a look a this one once again.

**Review by xSAVIKx — COMMENTED** [review]
@sasha-tkachev please fix the linter errors and take a look once again at the docs suggestion.

File: cloudevents/http/event.py (hunk 3 above)
Comment (sasha-tkachev) [reply cloudevents/http/event.py]: @xSAVIKx  Sorry, completely missed the suggested change.

**Comment by xSAVIKx** [comment]
@sasha-tkachev can you please update your branch as well as make sure that the linter is happy? You can do that by running `tox` locally (`tox -e reformat`)

**Comment by xSAVIKx** [comment]
@sasha-tkachev your branch is still outdated. You need to merge upstream master into yours.

**Comment by sasha-tkachev** [comment]
got it

**Review by xSAVIKx — APPROVED** [review]

**Pull request merged**